apps/desktop/src/components/PasswordInput.tsx

```diff
@@ -6,22 +6,25 @@ import {
   type InputProps,
   InputRightElement,
 } from "@chakra-ui/react";
+import { usePasswordValidation } from "@umami/components";
 import { useState } from "react";
 import { type FieldValues, type Path, type RegisterOptions, useFormContext } from "react-hook-form";
 
 import { EyeIcon, EyeSlashIcon } from "../assets/icons";
+import colors from "../style/colors";
 
-const MIN_LENGTH = 8;
+const MIN_LENGTH = 12;
 
 // <T extends FieldValues> is needed to be compatible with the useForm's type parameter (FormData)
 // <U extends Path<T>> makes sure that we can pass in only valid inputName that exists in FormData
 type PasswordInputProps<T extends FieldValues, U extends Path<T>> = {
   inputName: U;
   label?: string;
   placeholder?: string;
+  checkPasswordStrength?: boolean;
```

Comment:
nit: maybe `isCheckingPasswordStrength` here and in the type above to avoid confusion with a function?